code/mob/living/silicon/robot.dm

```diff
@@ -2333,7 +2332,8 @@
 			else	//Someone passed us an unkown modifier
 				message = "UNKNOWN ERROR: [src] in [myarea]"
 
-		if (message && mailgroup && radio_connection)
+		for(var/mailgroup in mailgroups)
+			//if (message && mailgroup && radio_connection)
```

Comment:
Clean up comment. I also think you might still want the `if (message)` check before the for-loop (please check, packet code is not my strong suit).